### ISSUE TITLE
Clarification on the training data

### ISSUE THREAD

**Issue by versteckterRabe** [chenhaoxing/DeMamba#14]
Hi, thank you very much for your work. What data should one use as a a baseline to retrain and compare? As a training set should all the models in the table from Zeroscope to Opensora be used? What files should be used as validation set? All those in the GenVideo-Val.zip? If not what data should be used? Also would it be possible to have the code to test a trained network?
Thank you
